union_pooling/tests/unit/union_pooler_test.py

```diff
@@ -51,6 +51,19 @@ def testDecayPoolingActivation(self):
       self.assertEquals(expected[i], result[i])
 
 
+  def testDecayPoolingActivationSpecifiedDecayRate(self):
+    self.unionPooler._decayFunctionSlope = 10
+    self.unionPooler._poolingActivation = numpy.array([0, 10, 20, 30, 40],
+                                                      dtype=REAL_DTYPE)
+    expected = numpy.array([0, 0, 10, 20, 30])
+
+    result = self.unionPooler._decayPoolingActivation()
+
+    self.assertEquals(len(expected), len(result))
```

Comment:
Good point. 

I get a ValueError when I used this assert function. The issue is that the arrays are numpy and comparing numpy arrays with == is ambiguous (http://stackoverflow.com/questions/10062954/valueerror-the-truth-value-of-an-array-with-more-than-one-element-is-ambiguous). 

I found the following works: self.assertTrue(numpy.array_equal(expected, result))